### ISSUE TITLE
Feature request:  'concat' for Variable

### ISSUE THREAD

**Issue by Kontrakt** [pytorch/pytorch#601]
I actually found it here: https://github.com/pytorch/pytorch/blob/f8d4f980b3df669e2c70993580d9c5b340eca97f/torch/autograd/variable.py#L765-L769

Is there any particular reason for keeping it and and a few other functions hidden (some seem to be reimplemented by the way)?

**Comment by apaszke** [comment]
Don't call the function directly, use [`torch.cat([var1, var2, var3], dim)`](http://pytorch.org/docs/torch.html#torch.cat).
